homeassistant/components/honeywell/climate.py

```diff
@@ -39,6 +39,9 @@
 
 PRESET_HOLD = "Hold"
 
+HEATING_MODES = {"heat", "emheat", "auto"}
+COOLING_MODES = {"cool", "auto"}
```

Comment:
You could use `HVACMode.COOL.value` if you need the string variant

Reply:
I just went through this exercise on the late review PR- these are used to look for API responses, not from HA enum values.  (They are different in some cases)

Essentially these are the keys from HW_MODE_TO_HVAC_MODE that match the given mode.

Reply:
I guess you may have looked at the code before I resolved the conflicts?

Reply:
I suppose I did